### ISSUE TITLE
Encryption not working with nested object with its own "schema"

### ISSUE THREAD

**Issue by lafama** [cinch-financial/mongoose-encryption#8]
How would i encrypt the field account in the following model
```
var user=new Schema({
name:{type:String,default:''},
account:{
secretcode:{type:String,default:''},
anothersecretcode:{type:String,default:''
}
});
```

**Comment by joegoldbeck** [comment]
working on this in branch `nest` 2e89af62041eaddf74efc6baaaf87428d61e7f83

**Comment by joegoldbeck** [comment]
fixed in 0.13.0 929bc05c9855b87f10591edd0633f9775b18b24b
